client/components/CreateJob.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { theme } from '../src/constants';
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  background-color: ${theme.colors.surface};
+  height: 100vw;
+`;
+
+const Header = styled.h1`
+  margin: 20px;
+  display: flex;
+  justify-content: center;
+  color: ${theme.colors.accent};
+  font-family: sans-serif;
+`;
+
+const InnerContainer = styled.div`
+  background-color: ${theme.colors.surfaceLight};
+  min-width: 400px;
+  width: 20vw;
+  margin-top: 80px;
+  border-radius: 7px;
+  padding: 80px;
+  box-shadow: 0px 4px 16px rgba(0,0,0,0.2);
+  justify-content: center;
+  max-height: 560px;
+`;
+
+const Input = styled.input`
+  margin: 10px 0;
+  height: 54px;
+  width: 100%;
+`;
+
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  > * {
+    box-sizing: border-box;
+  }
+`;
+
+const Select = styled.select`
+  margin: 20px 0;
+  height: 30px;
+  width: 100%;
+
+`;
+
+const Submit = styled.button`
```

Comment:
If you want to make the button consistent with the login page - https://material-ui.com/components/buttons/